flang/test/Driver/mlink-builtin-bc.f90

```diff
@@ -0,0 +1,15 @@
+! Test -mlink-builtin-bitcode flag
+! RUN: %flang -emit-llvm -c -o %t.bc %S/Inputs/libfun.f90
+! RUN: %flang_fc1 -emit-llvm -o - -mlink-builtin-bitcode %t.bc %s 2>&1 | FileCheck %s
+
+! CHECK: define internal void @libfun_
+
+! RUN: not %flang_fc1 -emit-llvm -triple x86_64-unknown-linux-gnu -o - -mlink-builtin-bitcode %no-%t.bc %s 2>&1 | FileCheck %s --check-prefix=ERROR
```

Comment:
My guess is that this doesn't need a triple [and probably never did]? So to keep it "architecture independent", perhaps remove that?  It's nit-picky, but I think it's useful to avoid having any reference to architecture/OS things in the tests when they don't HAVE to.

Reply:
Yes, I forgot to remove this when I fixed the tests. Thanks for finding it.